tests/drivers/dma/loop_transfer/src/dma.c

```diff
@@ -86,39 +103,43 @@ void main(void)
 	dma_cfg.block_count = 1U;
 	dma_cfg.head_block = &dma_block_cfg;
 
-	chan_id = 0U;
-
 	printk("Starting the transfer and waiting for 1 second\n");
 	dma_block_cfg.block_size = strlen(tx_data);
 	dma_block_cfg.source_address = (u32_t)tx_data;
 	dma_block_cfg.dest_address = (u32_t)rx_data[transfer_count];
 
-	if (dma_config(dma, chan_id, &dma_cfg)) {
-		printk("ERROR: transfer config\n");
-		return;
-	}
+	for (chan_id = FIRST_DMA_CHANNEL;
+		chan_id <= LAST_DMA_CHANNEL;
```

Comment:
indentation issue (should be below last '(' + 1 space)

Reply:
done